app/models.py

```diff
@@ -0,0 +1,67 @@
+from werkzeug.security import generate_password_hash, check_password_hash
+
+
+class User(object):
+    def __init__(self, name, email, password):
+        self.name = name
+        self.email = email
+        self.__password = password
+
+@property
+def password():
+
+    raise AttributeError("password is not a readable attribute.")
+
+@password.setter
+def password(self, password):
+
+    self.password_hash = generate_password_hash(password)
+
+
+def verify_password(self, password):
```

Comment:
why this function ?? it should be an instance method of user function

Reply:
So should the function be under the User class as a method?

Reply:
Noted